nodeup/pkg/model/networking/common.go

```diff
@@ -30,8 +30,12 @@ var _ fi.NodeupModelBuilder = &CommonBuilder{}
 
 // Build is responsible for copying the common CNI binaries
 func (b *CommonBuilder) Build(c *fi.NodeupModelBuilderContext) error {
-	if err := b.AddCNIBinAssets(c); err != nil {
-		return err
+	if b.NodeupConfig.Networking.AmazonVPC == nil &&
+		b.NodeupConfig.Networking.Calico == nil &&
+		b.NodeupConfig.Networking.Cilium == nil {
```

Comment:
can we reuse the new logic in pkg/apis/kops/model/instance_group.go ? that way we dont have to remember to update the same conditional in two places.

Reply:
sure, done